app/models/post.rb

```diff
@@ -11,4 +11,7 @@ def update_post_counter
   def recent_comments
     comments.order(created_at: :desc).limit(5)
   end
+  validates :Title, presence: true, length: { in: 6..250 }
+  validates :comments_counter, presence: true, numericality: { only_integer: true }
+  validates :likes_counter, presence: true, numericality: { only_integer: true }
```

Comment:
- [ ] [REQUIRED] Great job @Kaweesi-Matia , However, a part of the project requirement states that the `CommentsCounter` must be an integer greater than or equal to zero, and also the `LikesCounter` must be an integer greater than or equal to zero. Kindly implement this as you have implemented it in the User model validation for the `PostCounters`.

Reply:
Implement validations as required by the reviewer